src/surfclass/rasterreader.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=R0916
```

Comment:
Hvad gør det her?

Reply:
Den slår reglen om Maximum number of boolean expressions in an if statement fra i denne fil. Der må være max fem, men mit check af window mod rasterstørrelse bruger otte, og jeg vil helst ikke lave krumspring der.

Reply:
Super 👍 Vidste ikke pylint læste header comments 🧠